test/functional/android-helper-e2e-specs.js

```diff
@@ -4,8 +4,12 @@ import helpers from '../../lib/android-helpers';
 import sampleApps from 'sample-apps';
 import ADB from 'appium-adb';
 
-let apiDemos = sampleApps('ApiDemos-debug');
-let appPackage = 'io.appium.android.apis';
+let opts = 
```

Comment:
according to our style guide, please make this:

```js
let opts = {
  localApkPath: sampleApps('ApiDemos-debug'),
  pkg: 'io.appium.android.apis',
  androidInstallTimeout: 90000
};
```

(note spacing, line breaks, indentation)